src/sidebar/media-embedder.js

```diff
@@ -100,6 +101,29 @@ var embedGenerators = [
     return null;
   },
 
+/** Match Internet Archive URLs
+ *  The patterns are:
+ *  1. https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3
+ *  2. https://archive.org/embed/WHDH_20151121_043400_The_Tonight_Show_Starring_Jimmy_Fallon?start=360&end=420.3
```

Comment:
These two patterns are the same. @judell did you mean for the second one to be `/details/...`?

Reply:
Oops. Yeah. Never trust the documentation...